native/src/components/PoiListItem.tsx

```diff
@@ -1,33 +1,79 @@
-import * as React from 'react'
-import { ReactNode } from 'react'
+import React, { memo, ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
 import styled from 'styled-components/native'
 
-import { PoiModel } from 'api-client'
+import { PoiFeature } from 'api-client/src'
 import { ThemeType } from 'build-configs'
 
-import ListItem from './ListItem'
+import EventPlaceholder1 from '../assets/EventPlaceholder1.jpg'
+import { contentDirection } from '../constants/contentDirection'
+import { ListItemViewPropsType } from './ListItem'
+import SimpleImage from './SimpleImage'
 
-type PropsType = {
-  poi: PoiModel
+const Distance = styled.Text`
+  color: ${props => props.theme.colors.textColor};
+  font-family: ${props => props.theme.fonts.native.contentFontRegular};
+  margin-top: 4px;
+`
+const Thumbnail = styled(SimpleImage)`
+  width: 100px;
+  height: 100px;
+  flex-shrink: 0;
+  border-radius: 5px;
+`
+
+const ListItemView = styled.View<ListItemViewPropsType>`
+  flex: 1;
+  flex-direction: ${props => contentDirection(props.language)};
+  padding: 24px 0;
+`
+const StyledTouchableOpacity = styled.TouchableOpacity`
+  flex: 1;
+  flex-direction: column;
+  border-bottom-width: 1px;
+  border-bottom-color: #585858;
+`
+
+const Description = styled.View`
+  flex: 1;
+  height: 100%;
+  flex-direction: column;
+  flex-grow: 1;
+  font-family: ${props => props.theme.fonts.native.decorativeFontRegular};
+  padding: 0 32px;
+  justify-content: center;
+`
+const Title = styled.Text`
+  font-weight: 700;
+  font-family: ${props => props.theme.fonts.native.decorativeFontBold};
+  color: ${props => props.theme.colors.textColor};
+`
+
+type PoiListItemProps = {
+  poi: PoiFeature
   language: string
   navigateToPoi: () => void
   theme: ThemeType
 }
-const Description = styled.Text`
-  color: ${props => props.theme.colors.textColor};
-  font-family: ${props => props.theme.fonts.native.contentFontRegular};
-`
 
-class PoiListItem extends React.PureComponent<PropsType> {
-  render(): ReactNode {
-    const { poi, language, navigateToPoi, theme } = this.props
-    const { thumbnail } = poi
+export const PoiListItem: React.FC<PoiListItemProps> = memo(
```

Comment:
Was there an explicit reason why you memoed this component?

Reply:
ListItem should only be rerendered if props change. this is the equivalent to PureComponent